packages/core/docs/commercetools/configuration.md

```diff
@@ -9,180 +9,71 @@ Commercetools configuration is located in two places:
-      authHost: 'https://auth.sphere.io',
-      projectKey: '<PROJECT_KEY>',
-      clientId: '<CLIENT_ID>',
-      clientSecret: '<CLIENT_SECRET>',
-      scopes: [
-        'manage_products:<PROJECT_KEY>',
-        /* other scope rules */
-      ]
-    },
-    currency: 'USD',
-    country: 'US'
-  }
-}
-```
-
-### uri
-
-Link to your Commercetools GraphQL API instance. It should look like this:
-`https://<DOMAIN_NAME>.com/<PROJECT_KEY>/graphql`
-
-### authHost
-
-Link to Commercetools Authentication Server. It is used to request an access token from commercetools OAuth 2.0 service. To choose the nearest service, please visit [Commercetools hosts list](https://docs.commercetools.com/api/authorization)
-
-### projectKey
-
-i.e. `my-awesome-vsf-project`
-
-### clientId
-
-Unique Commercetools Client ID. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
-
-### clientSecret
-
-Commercetools secret API key. Visit [Commercetools documentation](https://docs.commercetools.com/tutorials/getting-started#creating-an-api-client) for more details about creating an API Client
-
-### scopes
-
-The scope constrains the endpoints to which a client has access, and whether a client has read or write access to an endpoint. Visit [Commercetools documentation](https://docs.commercetools.com/api/scopes#top) for more details about Scopes.
-
-
-Full configuration:
-
-```js
-interface Config<T = any> {
-  client?: ApolloClient<T>;
-  api: ApiConfig;
-  customOptions?: ApolloClientOptions<any>;
-  currency: string;
-  locale: string;
-  country: string;
-  countries: LocaleItem[];
-  currencies: LocaleItem[];
-  locales: LocaleItem[];
-  languageMap: Record<string, any>;
-  acceptLanguage: string[];
-  cookies: CookiesConfig;
-  auth?: Auth;
-  forceToken?: boolean;
-  handleIsTokenUserSession: (token: Token) => boolean;
-}
-```
-
-#### `client`
-
-Connection to the Commercetools client. `SdkAuth` and `TokenProvider` are imported from `@commercetools/sdk-auth`
-
-```js
-interface ClientInstance extends ApolloClient<any> {
-  sdkAuth?: SdkAuth;
-  tokenProvider?: TokenProvider;
-}
-```
-
-#### `api`
-
-Commercetools API configuration from `middleware.config.js`
-
-```js
-interface ApiConfig {
-  uri: string;
-  authHost: string;
-  projectKey: string;
-  clientId: string;
-  clientSecret: string;
-  scopes: string[];
-}
-```
-
-#### `countries`, `currencies`, `locales`
+- `useNuxtI18nConfig` - when this variable is set to true, `@vue-storefront/commercetools/nuxt` package will use i18n config object provided in nuxt.config.js. When set to false, `i18n` config should be declared directly inside this package configuration. You can read more about `i18n` config in Vue Storefront [here](../advanced/internationalization.md)
 
-Array of accepted items
+If you do not have this package installed locally, please run:
```

Comment:
this part is obsolete imho

Reply:
This was a recommendation from @andrzejewsky. If you think it is not needed, I can remove it as well :)

Reply:
Please remove as this is not the only thing once should do to make vsf work and this usage scenario wasn'e even tested.